testsuite/features/init_clients/allcli_update_activationkeys.feature

```diff
@@ -53,7 +53,18 @@ Feature: Update activation keys
     And I check "SLE-Module-DevTools15-SP4-Pool for x86_64"
     And I check "SLE-Module-Containers15-SP4-Updates for x86_64"
     And I check "Fake-RPM-SLES-Channel"
-    And I click on "Update Activation Key"
+    Then I should see "SLE-Module-Basesystem15-SP4-Pool for x86_64" as checked
+    And I should see "SLE-Module-Basesystem15-SP4-Updates for x86_64" as checked
+    And I should see "SLE-Module-Server-Applications15-SP4-Pool for x86_64" as checked
+    And I should see "SLE-Module-Server-Applications15-SP4-Updates for x86_64" as checked
+    And I should see "SLE-Module-DevTools15-SP4-Pool for x86_64" as checked
+    And I should see "SLE-Module-DevTools15-SP4-Updates for x86_64" as checked
+    And I should see "SLE-Module-Desktop-Applications15-SP4-Pool for x86_64" as checked
+    And I should see "SLE-Module-Desktop-Applications15-SP4-Updates for x86_64" as checked
+    And I should see "SLE-Module-Containers15-SP4-Pool for x86_64" as checked
+    And I should see "SLE-Module-Containers15-SP4-Updates for x86_64" as checked
+    And I should see "Fake-RPM-SLES-Channel" as checked
```

Comment:
I think this doesn't make sense.
The reason is that we suppose to validate it already:

```
When(/^I check "([^"]*)"$/) do |identifier|
  check(identifier)
  raise "Checkbox #{identifier} not checked." unless has_checked_field?(identifier)
end
```